### ISSUE TITLE
Copying file into tmpDir is not working windows server

### ISSUE THREAD

**Issue by houzawak** [jjalvarezl/JjalvarezlPDFjsViewerBundle#5]
Copying file into tmpDir is not working windows server. i get the message: unknwn command **cp** .

**Comment by jjalvarezl** [comment]
Right, it seems that **cp** is only for Unix shell environment. I have not tested it on windows yet but it is an easy fix. Is just make differences about platforms on that command. I have not expected that could be needed on a Windows platform.
